usaspending_api/conftest_helpers.py

```diff
@@ -75,16 +72,24 @@ def update_index(self, load_index: bool = True, **options):
         for the index, and add contents.
         """
         self.delete_index()
-        self.client.indices.create(index=self.index_name, body=self.template)
+        call_command("es_configure", "--template-only", f"--load-type={self.index_type}")
+        self.client.indices.create(index=self.index_name)
         create_award_type_aliases(self.client, self.etl_config)
         create_load_alias(self.client, self.etl_config)
-        call_command("es_configure", "--load-type", self.index_type)
-        # max_result_window is not set on the index until its settings are updated with es_configure command
-        idx_settings = self.client.indices.get_settings(index=self.index_name)
-        self.etl_config["max_query_size"] = int(idx_settings[self.index_name]["settings"]["index"]["max_result_window"])
+        self.etl_config["max_query_size"] = self._get_max_query_size()
         if load_index:
             self._add_contents(**options)
 
+    def _get_max_query_size(self):
+        upper_name = ""
+        if self.index_type == "award":
+            upper_name = "AWARDS"
+        elif self.index_type == "covid19-faba":
+            upper_name = "COVID19_FABA"
+        elif self.index_type == "transaction":
+            upper_name = "TRANSACTIONS"
```

Comment:
Having this IF statement and the near-identical one below is a code smell. We'll leave for future refactoring since this PR is only to fix an ETL problem